graphics/src/SkeletonAnimation.cc

```diff
@@ -45,6 +45,8 @@ SkeletonAnimation::SkeletonAnimation(const std::string &_name)
 //////////////////////////////////////////////////
 SkeletonAnimation::~SkeletonAnimation()
 {
+  for (auto &it : this->data->animations)
+    delete it.second;
```

Comment:
These could also be replaced by smart pointers.

Reply:
yep done. c4c3a86